app/models/event.rb

```diff
@@ -0,0 +1,10 @@
+class Event < ApplicationRecord
+
+    scope :upcoming, -> { where("event_date >= ?", Date.today) }
+    scope :past, -> { where("event_date < ?", Date.today) }
+
+
+    belongs_to :creator, class_name: "User"
+    has_many :attended_events, foreign_key: "attended_event_id"
```

Comment:
Related to my comment in line 4 of `user.rb`, this naming of the association got me confused on why an event can attend an event? I would mistakenly think that `birthday.attended_events` means that my birthday celebration can attend another event.